app/views/users/_pending.html.erb

```diff
@@ -0,0 +1,8 @@
+<% @pending_request.each do |p| %>
+  <% if @user == current_user %>
+    <%= p.name.capitalize %> wants to be your friend
+    <div class="friend-btn">
+      <%= confirm_button(p) %>
+    </div>
+  <% end %>
```

Comment:
Please remove all logic from your views. Move them to helpers.

**IMPORTANT!** I am only pointing this out here. Please go through your views and refactor the code as necessary.